From 1c9ffef4d85327fe517bdcf7e11c339a4862ba2e Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Wed, 12 Jun 2019 21:21:34 +0200
Subject: [PATCH] CMakeLists.txt: remove NONE keyword for languages from
 project definition. This solves a bug where SoQt is not found on Gentoo
 systems (see https://github.com/waebbl/waebbl-gentoo/issues/42)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Thanks to QÍN Yuhào for coming up with this fix.

Suggested-by: QÍN Yuhào <qinyh18@mails.tsinghua.edu.cn>
Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 76d0d06..48bd3c0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-project(pivy_cmake_setup NONE)
+project(pivy_cmake_setup)
 cmake_minimum_required(VERSION 3.5)
 
 
-- 
2.21.0

